kafka-on-docker/README.md

```diff
@@ -0,0 +1,145 @@
+<!-- title: How to run Kafka locally on Docker -->
```

Comment:
may just be me but i'm used to saying / seeing "run in Docker" or "run with Docker". just a suggestion
```suggestion
<!-- title: How to run Kafka locally with Docker -->
```